teacher.rb

```diff
@@ -0,0 +1,11 @@
+require_relative 'person'
+class Teacher < Person
+  def initialize(age, specialization, parent_permission: true, name: 'Unknown')
+    super(id, age, parent_permission: parent_permission, name: name)
+    @specialization = specialization
```

Comment:
[ Recommendation change ]

- [ ] You could remove the id you are passing in the super() as it's not being received for this class. The id is auto-generated in the Person class and is not expected to be inputted when a teacher is created. It's okay to remove it from the super. 👍🏽